spec/index.bs

```diff
@@ -515,6 +522,7 @@ dictionary GPUBindGroupLayoutBinding {
     required GPUShaderStageFlags visibility;
     required GPUBindingType type;
     GPUTextureViewDimension textureDimension;
```

Comment:
side note: should this be required or default to 2d?

Reply:
It should probably default to 2d. I'll open another PR for it.